Cargo.toml

```diff
@@ -9,5 +9,9 @@ repository = "https://github.com/mrhooray/kdtree-rs"
 documentation = "https://docs.rs/kdtree"
 license = "MIT OR Apache-2.0"
 
+[dependencies]
+failure = "~0.1"
+failure_derive = "~0.1"
```

Comment:
Tilde Requirements like these are questionable. Use just 0.1 if you want better semver compatibility.

Reply:
Sorry, was just copying the tilde use in `dev-dependencies`. Updated and force-pushed.